search-api/src/search_api/models/errors.py

```diff
@@ -0,0 +1,5 @@
+class DbRowNotFound(Exception):
```

Comment:
put this in the exceptions file with all the other ones using the base exception pls